runtime/terms.go

```diff
@@ -97,15 +97,15 @@ func newTerms(fpath, desc string) (*terms, error) {
 
 func (t *terms) String() string { return t.desc }
 
-func (t *terms) eval() (interface{}, error) {
```

Comment:
The unparam issue here was that that `error` would always return `nil`.  So I've done some refactoring here and downstream to remove the error parameter.

Please consider this carefully and let me know if we should be tackling the lint issue differently.